src/sentry/utils/http.py

```diff
@@ -19,6 +20,12 @@
 ParsedUriMatch = namedtuple('ParsedUriMatch', ['scheme', 'domain', 'path'])
 
 
+class FilterTypes(object):
```

Comment:
i'm not totally sure where this and the other two functions you added should go just in terms of code organization. maybe you could just make a separate `utils/data_filters.py`? otherwise, lgtm